source/OutfitInfoDisplay.cpp

```diff
@@ -458,10 +458,13 @@ void OutfitInfoDisplay::UpdateAttributes(const Outfit &outfit)
 			}
 	}
 
-	bool isContinuous = (reload <= 1);
+	bool isContinuous = (reload <= 1. && outfit.TotalLifetime() == 1.);
+	bool isContinuousBurst = (outfit.BurstCount() > 1 && outfit.BurstReload() <= 1. && outfit.TotalLifetime() == 1.);
```

Comment:
Weapon::TotalLifetime isn't a simple getter, so avoid calling it multiple times.
```suggestion
	bool oneFrame = (outfit.TotalLifetime() == 1.);
	bool isContinuous = (reload <= 1. && oneFrame);
	bool isContinuousBurst = (outfit.BurstCount() > 1 && outfit.BurstReload() <= 1. && oneFrame);
```